test/tests/functional/pbs_hook_set_jobenv.py

```diff
@@ -0,0 +1,976 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2016 Altair Engineering, Inc.
```

Comment:
Please update the copyright year number to 2017.

Reply:
Done.